Test/Framework/Visual/MiscellaneousTests.cs

```diff
@@ -82,6 +82,9 @@ namespace MonoGame.Tests.Visual {
 	class MiscellaneousTests : VisualTestFixtureBase
     {
 		[Test]
+#if XNA
+        [Ignore]
+#endif
```

Comment:
I am a little concerned about skipping this test as I don't know why we would be.  Hopefully we can work this out later.